### DISCUSSION TITLE
docs/v1.10.x/reference/underlay-topology/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1930] [Announcements]
# docs/v1.10.x/reference/underlay-topology/

https://kubeovn.github.io/docs/v1.10.x/reference/underlay-topology/

**Comment by xujunjie-cover** [comment]
无 Vlan Tag 下总览和多 VLAN 总览应该是写反了- -

**Reply by oilbeater** [reply]
已修改，多谢
